src/collections/_documentation/platforms/javascript/sourcemaps/troubleshooting.md

```diff
@@ -48,9 +48,9 @@ or if your file is similar to:
 then your uploaded artifact should also be named `https://example.com/dist/js/script.min.js.map`
 
 
-### Verify artifact names and file path are correct
+### Verify artifact names match stack trace frames
 
-If you’ve uploaded source maps and they aren’t applying to your code in an issue in Sentry, take a look at the filename in the stack trace frames, for example, `/scripts/script.min.js`. You can also open up the JSON of the event and look for the `abs_path` to see exactly where we’re attempting to resolve the file  - i.e. `http://localhost:8000/scripts/script.js`
+If you’ve uploaded source maps and they aren’t applying to your code in an issue in Sentry, take a look at the filename in the stack trace frames, for example, `/scripts/script.min.js`. You can also open up the JSON of the event and look for the `abs_path` to see exactly where we’re attempting to resolve the file  - i.e. `http://localhost:8000/scripts/script.js`. This value needs to match the uploaded artifact names.
```

Comment:
Good. Maybe one more sentence tying in the example you're giving, so like:

"This value needs to match the uploaded artifact names. So, continuing our example, your uploaded source map should be called either <options here which will work>. (Same goes for the corresponding minified and original javascript files.)"